packages/browser-wallet-api-helpers/README.md

```diff
@@ -142,7 +142,12 @@ It is possible to sign arbitrary messages using the keys for an account stored i
 
 If the wallet is locked, or you have not connected with the wallet (or previously been whitelisted) or if the user rejects signing the meesage, the `Promise` will reject.
 
-The following exemplifies requesting a signature of a message:
+The message should either utf8 string or an object with the following fields:
+
+-   data: A hex string representing the bytes that should be signed.
+-   schema: A base64 string that represents a schema for the data field, and which can be used to deserialize the data into a JSON format.
```

Comment:
What kind of schema? Can we link to somewhere with some help for this? I wouldn't know where to start if I wanted to use this schema functionality. Is it the smart contract schema stuff?

Reply:
I've added "(For reference, the type of schema used here is the same that is used for smart contract types)"
But I'm not sure where would we link them, but this feature is used for sponsored transactions, so we should have an explanation for this.

Reply:
https://developer.concordium.software/en/mainnet/smart-contracts/general/contract-schema.html